.github/workflows/coverage.yml

```
@@ -128,32 +149,44 @@ jobs:
             STATUS="ok"
             COVERAGE="0.0"
 
-            if [ -n "$INCLUDED_PACKAGES" ] && go test -coverprofile=coverage_raw.out $INCLUDED_PACKAGES; then
-              cp coverage_raw.out coverage.out
-
-              # Remove excluded files from coverage
-              FILE_COUNT=$(echo "$EXCLUDE_FILES" | yq 'length')
-              for j in $(seq 0 $((FILE_COUNT - 1))); do
-                FILE=$(echo "$EXCLUDE_FILES" | yq ".[$j]" | tr -d '"')
-                echo "    Removing $FILE from coverage"
-                sed -i "/$FILE/d" coverage.out || true
-              done
+            if [ -n "$INCLUDED_PACKAGES" ]; then
+              echo "    Running tests with coverage..."
+              set +e  # Don't exit on test failures
+              go test -coverprofile=coverage_raw.out $INCLUDED_PACKAGES
+              TEST_EXIT_CODE=$?
+              set -e
```

Is this required because our tests are too unreliable? I'm worried it may lead to silently ignored failures.

---

Included that to ignore errors occured due to no test files in the packages. But modified the code now to filter out those packages.